src/main/java/uk/gov/hmcts/ccd/domain/model/definition/CaseDetails.java

```diff
@@ -208,6 +215,22 @@ public void setLastStateModifiedDate(LocalDateTime lastStateModifiedDate) {
         this.lastStateModifiedDate = lastStateModifiedDate;
     }
 
+    public Map<String, JsonNode> getSupplementaryData() {
+        return supplementaryData;
+    }
+
+    public void setSupplementaryData(Map<String, JsonNode> supplementaryData) {
+        this.supplementaryData = supplementaryData;
+    }
```

Comment:
we don't want to allow getting and setting the whole supplementary data. Consider providing methods to instead get and set specific properties

Reply:
it is work in progress I added for my own benefit while I am doing the work . But understood